suite-native/settings/src/settingsSlice.ts

```diff
@@ -12,6 +12,7 @@ export interface AppSettingsState {
     isFirmwareHashCheckEnabled: boolean;
     areTestnetsEnabled: boolean;
     shouldShowAutoEjectAlert: boolean;
+    disconnectedDevicesDuringDiscovery: Device['id'][];
```

Comment:
Is it necessary to introduce a state for this? What about using the discovery state? Checking if it's failed with errorCode `Device_Disconnected`? 

If we really need the list of devices disconnected during discovery in the state, it doesn't belongs to settings slice and we need to handle deleting the device reference on "eject"

Reply:
Maybe even better condition would be if selected device is not connected and the discovery is not completed? That would work even for the case when you enable more coins while the device is disconnected.